.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      matrix:
+        php: ['7.3', '7.4']
+        sf_version: ['4.4.*', '5.0.*']
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.1.0
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+          tools: flex
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download dependencies
+        env:
+          SYMFONY_REQUIRE: ${{ matrix.sf_version }}
+        run: composer update --no-interaction --prefer-dist --optimize-autoloader --prefer-stable
+
+      - name: Run tests
+        run: ./vendor/bin/phpunit
+
+  lowest:
```

Comment:
you can put `--prefer-lowest` into the matrix as well by using something like:
```
strategy:
  matrix:
    # [...]
    composer_flags: ['', '--prefer-lowest']
steps:
  # [...]
  - run: composer update --no-interaction --prefer-dist --optimize-autoloader --prefer-stable ${{ matrix.composer_flags }}
```
This would test the lowest symfony 5.0 version as well.

Reply:
And you don't need a separate test block.

Reply:
Im more happy with the prefer-lowest like this. 

Putting it in the matrix means that we will run `prefer-lowest` for each PHP version and Symfony version. That is too many builds we need to wait for. =)